processor/src/decorators/adv_stack_injectors.rs

```diff
@@ -338,6 +338,37 @@ where
 
         Ok(())
     }
+
+    /// Pushes values onto the advice stack which are required for verification of a DSA in Miden VM.
+    ///
+    /// Inputs:
+    ///   Operand stack: [PK, MSG, ...]
+    ///   Advice stack: [...]
+    ///   Advice map: {PK: [SK]}
+    ///
+    /// Outputs:
+    ///   Operand stack: [PK, MSG, ...]
+    ///   Advice stack: [DATA]
+    ///   Advice map: {PK: [SK]}
+    ///
+    /// Where:
+    /// - PK is the digest of an expanded public.
+    /// - MSG is the digest of the message to be signed.
+    /// - DATA is the needed data for signature verification in the VM.
```

Comment:
I would probably define `DATA` in a bit more detail (I believe there are several components to it).

Reply:
But they are very specific to Falcon signature. For other DSA they could be group elements etc.

Reply:
Ah - makes sense! Let's leave it as is.